bin/before_install

```diff
@@ -18,3 +18,18 @@ elif [ ! -d "$spec_manageiq" ]; then
   echo "== Cloning manageiq sample app =="
   git clone https://github.com/ManageIQ/manageiq.git --branch oparin --depth 1 "$spec_manageiq"
 fi
+
+if [ -n "$CI" ]; then
+  echo "== Repository Owner: $GITHUB_REPOSITORY_OWNER =="
+  echo "== Base Ref: $GITHUB_BASE_REF =="
+  echo "== Ref Name: $GITHUB_REF_NAME =="
+
+  # Gemfile.lock.release only applies to non-master branches and PRs to non-master branches
+  if [[ "$GITHUB_REPOSITORY_OWNER" = "ManageIQ" && "$GITHUB_BASE_REF" != "master" && "$GITHUB_REF_NAME" != "master" && "$GITHUB_REF_NAME" != "dependabot/"* ]]; then
+    echo "== Setup Gemfile.lock.release =="
+    cp -f "$spec_manageiq/Gemfile.lock.release" "$gem_root/Gemfile.lock"
+
+    `$spec_manageiq/bin/ci/patch_plugin_gemfile_lock`
```

Comment:
Why is this wrapped in backticks?  Since this is just a script, we should be able to just call it directly.